src/main/java/org/zalando/nakadi/domain/Permission.java

```diff
@@ -43,11 +43,26 @@ public boolean equals(final Object o) {
         final Permission that = (Permission) o;
         return Objects.equals(resource, that.resource) &&
                 Objects.equals(operation, that.operation) &&
-                Objects.equals(authorizationAttribute, that.authorizationAttribute);
+                Objects.equals(authorizationAttribute.getDataType(), that.authorizationAttribute.getDataType()) &&
+                Objects.equals(authorizationAttribute.getValue(), that.authorizationAttribute.getDataType());
```

Comment:
I guess the value should be compared to  value 
```suggestion
                Objects.equals(authorizationAttribute.getValue(), that.authorizationAttribute.getValue());
```